crates/shared/src/price_estimation.rs

```diff
@@ -152,31 +152,22 @@ impl NativePriceEstimators {
         Self(Vec::new())
     }
 
-    pub fn as_slice(&self) -> &[NativePriceEstimator] {
+    pub fn as_slice(&self) -> &[Vec<NativePriceEstimator>] {
         &self.0
     }
 }
 
 impl Default for NativePriceEstimators {
     fn default() -> Self {
-        Self(vec![NativePriceEstimator::GenericPriceEstimator(
+        Self(vec![vec![NativePriceEstimator::GenericPriceEstimator(
             PriceEstimatorKind::Baseline,
-        )])
+        )]])
     }
 }
 
 impl Display for NativePriceEstimators {
     fn fmt(&self, f: &mut Formatter) -> fmt::Result {
-        let mut it = self.as_slice().iter();
-        if let Some(kind) = it.next() {
-            write!(f, "{kind:?}")?;
-            for kind in it {
-                write!(f, ",{kind:?}")?;
-            }
-            Ok(())
-        } else {
-            f.write_str("None")
-        }
+        write!(f, "{:?}", self.as_slice())
```

Comment:
I think this is wrong - `Display` should be the mirror image as `FromStr` (this is the expectation of `clap`, where it uses `Display` to display things like default or selected values, and `FromStr` to parse values, so if you don't make them match, then `clap` will print in a format that is not accepted by the parser in a context where it is trying to display how to set a value).